projects/systelab-components/src/lib/breadcrumb/breadcrumb.component.ts

```diff
@@ -20,6 +20,8 @@ export class BreadcrumbComponent {
 	@Input() public items: BreadcrumbItem[];
 	@Input() public backgroundColor: string;
 	@Input() public fontColor: string;
+	@Input() public itemMaxWidth: '9999px';
```

Comment:
I am not familiar with this implicit type syntax. I would expect to see here: 
`itemMaxWidth: string = '9999px'`;

Reply:
The value after the @Input() must be a type and you're trying to add a default value. The way to do that is @Input() public itemMaxWidth: string = '9999px' or directly @Input() public itemMaxWidth = '9999px' because of the type is interfered from the default value